.github/workflows/php-coverage-codecov.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     name: Code Coverage
     runs-on: ubuntu-latest
     steps:
-      - uses: zenstruck/.github@php-coverage-codecov
+      - uses: ./.github/actions/php-coverage-codecov
```

Comment:
It's my understanding that we don't need to suffix with `@main` here.

Reply:
I think it will take the same branch/version on which the callable workflow was called